src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonDrawer/components/PolygonReviewButtons.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+
+import Button from "@/components/elements/Button/Button";
+import Menu from "@/components/elements/Menu/Menu";
+import StepProgressbar from "@/components/elements/ProgressBar/StepProgressbar/StepProgressbar";
+import Text from "@/components/elements/Text/Text";
+import { IconNames } from "@/components/extensive/Icon/Icon";
+import { useSitePolygonData } from "@/context/sitePolygon.provider";
+
+const PolygonReviewButtons = ({
+  openFormModalHandlerAddPolygon,
+  downloadSiteGeoJsonPolygons,
+  openFormModalHandlerSubmitPolygon,
+  record,
+  openFormModalHandlerUploadImages
+}: {
+  openFormModalHandlerAddPolygon: () => void;
+  downloadSiteGeoJsonPolygons: (uuid: string) => void;
+  openFormModalHandlerSubmitPolygon: () => void;
+  record: { uuid: string };
+  openFormModalHandlerUploadImages: () => void;
+}) => {
+  const context = useSitePolygonData();
+  const { toggleUserDrawing } = context || {};
+
+  const addMenuItems = [
+    {
+      id: "1",
+      render: () => <Text variant="text-12-bold">Create Polygons</Text>,
+      onClick: () => toggleUserDrawing?.(true)
+    },
+    {
+      id: "2",
+      render: () => <Text variant="text-12-bold">Add Polygon Data</Text>,
+      onClick: openFormModalHandlerAddPolygon
+    },
+    {
+      id: "3",
+      render: () => <Text variant="text-12-bold">Upload Images</Text>,
+      onClick: openFormModalHandlerUploadImages
+    }
+  ];
+
+  const polygonStatusLabels = [
+    { id: "1", label: "Draft" },
+    { id: "2", label: "Awaiting Approval" },
+    { id: "3", label: "Needs More Information" },
+    { id: "4", label: "Planting In Progress" },
+    { id: "5", label: "Approved" }
+  ];
+
+  return (
+    <div className="flex items-start gap-3">
+      <div className="w-full">
+        <div className="mb-2">
+          <Text variant="text-16-bold" className="mb-2 text-darkCustom">
+            Polygon Review
+          </Text>
+          <Text variant="text-14-light" className="text-darkCustom">
+            Add, remove or edit polygons that are associated to a site. Polygons may be edited in the map below;
+            exported, modified in QGIS or ArcGIS and imported again; or fed through the mobile application.
+          </Text>
+        </div>
+        <div className="flex gap-3">
+          <Menu menu={addMenuItems} className="flex-1">
+            <Button
+              variant="sky-page-admin"
+              className="h-fit w-full whitespace-nowrap"
+              iconProps={{
+                className: "w-4 h-4",
+                name: IconNames.PLUS_PA // Replace IconNames.PLUS_PA with the actual icon name
```

Comment:
Will this todo be completed as part of this PR? Same question on line 82 below

Reply:
Comments removed because it was from an older version.